installer/dev/main.cpp

```diff
@@ -31,6 +31,19 @@ int wmain(int argc, wchar_t *argv[])
         }
     }
 
-    return ProjectReunionInstaller::DeployPackages(quiet);
+    const HRESULT deployPackagesResult = ProjectReunionInstaller::DeployPackages(quiet);
```

Comment:
NIT: `const HRESULT deployPackagesResult{ ProjectReunionInstaller::DeployPackages(quiet) };`

[ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)